.github/workflows/static_build.yml

```diff
@@ -138,15 +134,7 @@ jobs:
           environment-name: mambabuild
           init-shell: bash cmd.exe
           create-args: >-
-            cli11>=2.2,<3
-            cpp-expected
-            nlohmann_json
-            simdjson-static>=3.3.0
-            spdlog
-            fmt
-            yaml-cpp-static>=0.8.0
-            libsolv-static>=0.7.24
-            reproc-cpp-static>=14.2.4.post0
+            cli11>=2.2,<3 cpp-expected nlohmann_json simdjson-static>=3.3.0 spdlog fmt yaml-cpp-static>=0.8.0 libsolv-static>=0.7.24 reproc-cpp-static>=14.2.4.post0
```

Comment:
Why not leave it as it is? It's more readable and it complies with the the examples in [setup-micromamba](https://github.com/mamba-org/setup-micromamba).

Reply:
Oh this might have been my yaml formatter - sorry for that!

Reply:
Fixed it